pages/common/traceroute.md

```diff
@@ -18,6 +18,6 @@
 
 `traceroute -q 5 {{host}}`
 
-- Specify size in bytes of datagram:
+- Specify size in bytes of probing packet:
 
-`traceroute {{host}} {{size<=32768}}`
+`traceroute {{host}} 42`
```

Comment:
This is still a token word. It should be `{{42}}`. https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax